docs/standard/library-guidance/nuget.md

```diff
@@ -86,13 +86,22 @@ NuGet packages with a version suffix are considered [pre-release](/nuget/create-
 
 ## Symbol packages
 
-NuGet supports [generating a separate symbol package](/nuget/create-packages/symbol-packages) containing debug PDB files alongside the main package containing .NET assemblies. The idea of symbol packages is they're hosted on a symbol server and are only downloaded by a tool like Visual Studio on demand.
+Symbol files (`*.pdb`) are produced by the .NET compiler alongside assemblies. Symbol files map execution locations to the original source code so you can step through source code as it is running using a debugger. NuGet supports [generating a separate symbol package](/nuget/create-packages/symbol-packages) containing symbol files alongside the main package containing .NET assemblies. The idea of symbol packages is they're hosted on a symbol server and are only downloaded by a tool like Visual Studio on demand.
 
-Currently the main public host for symbols - [SymbolSource](http://www.symbolsource.org/) - doesn't support the portable PDBs created by SDK-style projects and symbol packages aren't useful.
+Currently the main public host for symbols - [SymbolSource](http://www.symbolsource.org/) - doesn't support the portable symbol files created by SDK-style projects and symbol packages aren't useful. Until there is a recommended host for symbol packages, symbol files can be embedded in the main NuGet package. If you are building your NuGet package using an SDK-style project you can embed symbol files by setting the `AllowedOutputExtensionsInPackageBuildOutputFolder` property: 
```

Comment:
- nit unrelated to your PR: there should be a comma before "and symbol packages aren't useful"
- The relationship between symbol files and portable symbol files isn't clear here -- possibly the first sentence of line 89 should read "Portable symbol files". Or "the first "portable symbol files" on line 91 should include "(.pdb files)".